index.js

```diff
@@ -0,0 +1,68 @@
+function openNav() {
+  document.getElementById('myNav').style.width = '100%';
+}
+function closeNav() {
+  document.getElementById('myNav').style.width = '0%';
+}
+openNav();
+closeNav();
+const speakerList = [
+  {
+    name: 'Solomon Gebreslassie',
+    image: 'assets/solomon.png',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+  {
+    name: 'Haftamu Desta',
+    image: 'assets/haftamu.jpg',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+  {
+    name: 'Haftamu mekonen',
+    image: 'assets/haftamu-m.png',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+  {
+    name: 'Berihu Tesfay',
+    image: 'assets/berihu.png',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+  {
+    name: 'Kinfe G.Gergis',
+    image: 'assets/kinf.png',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+  {
+    name: 'Tsegay Kidu',
+    image: 'assets/tsegay.png',
+    title: 'substation construction expert',
+    resume: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit dolore beatae officia nobis, dolores accusamus atque labore quas in consequuntur nostrum eum sint vel a fugit repellat minus doloremque temporibus',
+  },
+];
+
+const section = document.querySelector('.feature-speaker');
+
+window.onload = () => {
+  speakerList.map(speaker => {
+    const classDetail = document.createElement('div');
+    classDetail.classList.add('detail');
+    classDetail.insertAdjacentHTML('beforeend', `
+      <div class="profesionalist">
+        <div class="right">
+          <img src=${speaker.image} alt="picture of haftamu">
+        </div>
+        <div class="left">
+          <h2>${speaker.name}</h2>
+          <p class="title">${speaker.title}</p>
+          <p class="resume">${speaker.resume}</p>
+        </div>
+      </div>
+    `);
+    section.appendChild(classDetail);
```

Comment:
- [ ] As suggested by the previous reviewer, and as the original design looks, Kindly make sure on mobile, you have only a few numbers of speakers displayed and a `SEE MORE` button, once clicked the user will be able to see the remaining number of speakers.
`Hint: YOu may consider displaying them none at first and make a button when clicked they become visible...` :smile:

Reply:
Dear @christianbiring1  the SEE MORE  part is option in the project requirement and now I have to time to do so

Reply:
Sorry about that. The previous reviewer must have missed it